zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -651,11 +651,16 @@ trait Sql {
     val WidthBucket = FunctionDef[(Double, Double, Double, Int), Int](FunctionName("width bucket"))
 
     //string functions
+    val Ascii       = FunctionDef[String, Int](FunctionName("ascii"))
     val CharLength  = FunctionDef[String, Int](FunctionName("character length"))
+    val Concat      = FunctionDef[(String, String), String](FunctionName("concat"))
```

Comment:
I defined it as `FunctionDef[(String, String), String]` because Oracle version of `CONCAT` function is limited to only two input arguments